SpatialGDK/Source/SpatialGDK/Public/Schema/UnrealObjectRef.h

```diff
@@ -45,6 +45,16 @@ struct FUnrealObjectRef
 		return FString::Printf(TEXT("(entity ID: %lld, offset: %u)"), Entity, Offset);
 	}
 
+	FORCEINLINE FUnrealObjectRef GetLevelReference() const
+	{
+		if (Path->Equals(TEXT("PersistentLevel")))
```

Comment:
Is every stably named object ref guaranteed to have `PersistentLevel` as an outer (eventually)?

Reply:
As far as I can tell. Yes. It seems that every sublevel has a `PersistentLevel` somewhere

Reply:
Naming a subobject "PersistentLevel" will break this logic. We could work around this by somehow indicating whether an object ref is a level (additional field? but that would be expensive)
Alternatively, we can warn users if they call anything that isn't a level "PersistentLevel" when creating an object ref.

Reply:
I think this is something we should consider if it comes up. It seems like a pretty specific edge case